packages/cadl-ranch/src/coverage/scenario-manifest.ts

```diff
@@ -6,6 +6,8 @@ import { ScenarioLocation, ScenarioManifest } from "@azure-tools/cadl-ranch-cove
 import { getSourceLocation, normalizePath } from "@typespec/compiler";
 import { relative } from "path";
 
+const scenarioTypes = ["azure", "non-azure"];
```

Comment:
I think  `non-azure` is not a great name for this. My understanding is that this should have 0 mention of azure.

Maybe just `standard`